README.md

```diff
@@ -7,8 +7,9 @@
   * [About the code](#about-the-code)
   * [Pointers](#pointers)
 * [Installation](#installation)
-  * [System-wide](#system-wide)
-  * [Virtualenv](#virtualenv)
+  * [System-wide with pip](#system-wide-with-pip)
+  * [System-wide from source](#system-wide-from-source)
+  * [virtualenv](#virtualenv)
```

Comment:
Why did you switch to lower case here (and also below)? It's a change unrelated to the commit message and IMHO not necessary: the software is called `virtualenv` (lower case), but https://virtualenv.pypa.io/en/stable/ uses `Virtualenv` in header lines and menus.